Examples/options/geant_fullsim_fccee_pgun.py

```diff
@@ -83,15 +87,15 @@
 # PODIO algorithm
 from Configurables import PodioOutput
 out = PodioOutput("out",
-                  filename = "fccee_fullsim_pgun.root",
+                  filename = input_filename,
                    OutputLevel=INFO)
 out.outputCommands = ["keep *"]
 
 # ApplicationMgr
 from Configurables import ApplicationMgr
 ApplicationMgr( TopAlg = [geantsim, out],
                 EvtSel = 'NONE',
-                EvtMax   = 1,
```

Comment:
Let's keep this at 1 Event. This configuration is tested every time someone makes a pull request, it will take too long otherwise.